src/Sql/dbo/Stored Procedures/Organization_Create.sql

```diff
@@ -27,7 +27,7 @@
     @Gateway TINYINT,
     @GatewayCustomerId VARCHAR(50),
     @GatewaySubscriptionId VARCHAR(50),
-    @ReferenceId VARCHAR(50),
+    @ReferenceData VARCHAR(50),
```

Comment:
Needs to be max right?

Reply:
Wow, yes, sorry I missed that and good catch, thank you.